Jenkinsfile

```diff
@@ -91,6 +98,77 @@ void buildManifest(String variant_name, String bitbake_image) {
             }
         }
 
+        if (variant_name == "qemu-x86-64_nogfx") {
+            try {
```

Comment:
Perhaps all this could be a separate Jenkinsfile that we load and run if needed? Since it is such a different mode of operation I think that would make sense.

Reply:
That sounds like a good idea. For now this is only a WIP, but I will break it out to finish the user story.